### ISSUE TITLE
Feature request: Subvert support

### ISSUE THREAD

**Issue by PrimaMateria** [AckslD/muren.nvim#23]
tpope's subvert command is a blessing when working with Typescript. Would it be possible for muren to use it? 

https://github.com/tpope/vim-abolish#substitution

**Comment by AckslD** [comment]
Maybe the function used to do the search and replace could be configurable such that the user could eg choose to use the one your suggesting, without coupling muren to that plugin. What do you think?

**Comment by PrimaMateria** [comment]
Yes, that would be great. This would also allow using some other alternatives that could appear in the future.
